src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +46,41 @@
      * Андрей - 600
      * София - 900</p>
      */
+
+    static DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public boolean checkDate(LocalDate firstDate, LocalDate secondDate) {
+        return firstDate.compareTo(secondDate) <= 0;
```

Comment:
should we create this method?

Reply:
> should we create this method?

i can't realize code without this method
i tried to use isBefore and isAfter from LocalDate
but i cought an error